eclass/distutils-r1.eclass

```diff
@@ -215,22 +212,22 @@ _distutils_set_globals() {
 				;;
 			flit_scm)
 				bdep+='
-					dev-python/flit_scm[${PYTHON_USEDEP}]
+					>=dev-python/flit_scm-1.7.0[${PYTHON_USEDEP}]
 				'
 				;;
 			hatchling)
 				bdep+='
-					>=dev-python/hatchling-1.3.1[${PYTHON_USEDEP}]
+					>=dev-python/hatchling-1.8.1[${PYTHON_USEDEP}]
 				'
 				;;
 			jupyter)
 				bdep+='
-					>=dev-python/jupyter_packaging-0.12.0-r1[${PYTHON_USEDEP}]
+					>=dev-python/jupyter_packaging-0.12.3[${PYTHON_USEDEP}]
 				'
 				;;
 			maturin)
 				bdep+='
-					>=dev-util/maturin-0.12.20[${PYTHON_USEDEP}]
+					>=dev-util/maturin-0.13.3[${PYTHON_USEDEP}]
```

Comment:
Can drop the old `if has_version '>=dev-util/maturin-0.13'.. else` block with this.

Reply:
Done, thanks!